Compile failure with updated eckit

See https://github.com/MetOffice/mo-bundle/issues/933.  Updating eckit to 2.0.0 together with necessary updates to some other ECMWF repositories results in this compile failure in orca-jedi:

In file included from /home/users/david.davies/cylc-run/mo-bundle-933/share/mo-bundle/orca-jedi/src/tests/orca-jedi/test_geometry.cc:8:
/home/users/david.davies/cylc-run/mo-bundle-933/share/mo-bundle/orca-jedi/src/tests/orca-jedi/test_geometry.cc: In function 'void orcamodel::test::test_24(std::string&, int&, int)':
/home/users/david.davies/cylc-run/mo-bundle-933/share/mo-bundle/orca-jedi/src/tests/orca-jedi/test_geometry.cc:25:34: error: 'exists' is not a member of 'eckit::system::Library'
   25 |   EXPECT(eckit::system::Library::exists("atlas-orca"));

The eckit version cannot be updated for the time being as it requires an atlas upgrade.  I don't know whether this can be addressed independently of the eckit upgrade or not.

--- a/src/tests/orca-jedi/test_geometry.cc
+++ b/src/tests/orca-jedi/test_geometry.cc
@@ -7,6 +7,7 @@
 #include "eckit/mpi/Comm.h"
 #include "eckit/testing/Test.h"
 #include "eckit/exception/Exceptions.h"
+#include "eckit/system/LibraryManager.h"
 
 #include "oops/base/Variables.h"
 
@@ -22,7 +23,7 @@ namespace test {
 //-----------------------------------------------------------------------------
 
 CASE("test basic geometry") {
-  EXPECT(eckit::system::Library::exists("atlas-orca"));
+  EXPECT(eckit::system::LibraryManager::exists("atlas-orca"));
 
   eckit::LocalConfiguration config;
   std::vector<eckit::LocalConfiguration> nemo_var_mappings(5);

--- a/src/tests/orca-jedi/test_increment.cc
+++ b/src/tests/orca-jedi/test_increment.cc
@@ -9,6 +9,7 @@
 #include "eckit/mpi/Comm.h"
 #include "eckit/testing/Test.h"
 #include "eckit/exception/Exceptions.h"
+#include "eckit/system/LibraryManager.h"
 
 #include "oops/base/Variables.h"
 
@@ -26,7 +27,7 @@ namespace test {
 //-----------------------------------------------------------------------------
 
 CASE("test increment") {
-  EXPECT(eckit::system::Library::exists("atlas-orca"));
+  EXPECT(eckit::system::LibraryManager::exists("atlas-orca"));
 
   eckit::LocalConfiguration config;
   std::vector<eckit::LocalConfiguration> nemo_var_mappings(4);

